src/client/datascience/jupyter/jupyterVariables.ts

```diff
@@ -92,8 +91,8 @@ export class JupyterVariables implements IJupyterVariables {
         );
     }
 
-    private async getVariableHandler(notebook?: INotebook): Promise<IJupyterVariables> {
-        if (this.debuggerVariables.active && (!notebook || notebook.status === ServerStatus.Busy)) {
+    private async getVariableHandler(_notebook?: INotebook): Promise<IJupyterVariables> {
```

Comment:
This function is just local to this class so can just remove the parameter if we are not using it, no need to _. But at the same time is it ok that we are missing that (!notebook || notebook.status === ServerStatus.Busy) clause?

Reply:
It doesn't work for the ipykernel debugger, but It might be necessary for the old debugger. I need to check

Reply:
Thinking about it, the old stuff (webview notebooks and interactive window) won't be available starting next month, so that doesn't matter. The native interactive window seems to not be hooked up yet (I checked in main), I'll check that in the next iteration.

So I'll remove the parameter and its mentions.

Reply:
Not hooked up to the variables window? It works for me in the interactive window